plugins/rss/init.js

```diff
@@ -1153,7 +1153,8 @@ rTorrentStub.prototype.setrsssettings = function()
 
 rTorrentStub.prototype.getrssdetailsResponse = function(data)
 {
-	$("#rsslayout").text(String(data).replace(/<br\s*\/?>(\s*\n)?/gi, "\n"));
+	const doc = new DOMParser().parseFromString(String(data), 'text/html');
+	$("#rsslayout").text(doc.body.textContent || '');
```

Comment:
**Give this a try, it might work. It will sanitize the HTML document object then return it.**

const doc = new DOMParser().parseFromString(String(data), 'text/html');
$("#rsslayout").setHTML(new Sanitizer(doc.body));

Reply:
I'm getting an error with the ```$("#rsslayout").text(doc.body.textContent || '');```  line changed to ```$("#rsslayout").setHTML(new Sanitizer(doc.body)); ```, on a fresh clone from the master branch + this commit/pull request

```
	const doc = new DOMParser().parseFromString(String(data), 'text/html');
	$("#rsslayout").setHTML(new Sanitizer(doc.body)); 
```

> [28.12.2022 07:28:30] JS error: [http://192.168.1.226:643/ line 2 > injectedScript : 4068] ReferenceError: Sanitizer is not defined
> [28.12.2022 07:28:35] JS error: [http://192.168.1.226:643/ line 2 > injectedScript : 4068] ReferenceError: Sanitizer is not defined

> root@ubuntulocal:/var/www/html/ruTorrent# git reset --hard
> HEAD is now at af643910 Merge pull request #2408 from TrimmingFool/specify-doc-lang
> root@ubuntulocal:/var/www/html/ruTorrent# git pull origin master
> From https://github.com/Novik/ruTorrent
>  * branch              master     -> FETCH_HEAD
> Updating af643910..084a60ea
> Fast-forward
>  conf/config.php               |  4 +++-
>  php/addtorrent.php            |  7 +++++--
>  php/util.php                  |  6 +++---
>  plugins/check_port/action.php |  2 +-
>  plugins/erasedata/fin.sh      |  2 +-
>  plugins/unpack/unrar_dir.sh   | 37 ++++++++++++++++++++++++++++++++-----
>  plugins/unpack/unrar_file.sh  | 20 +++++++++++++++-----
>  7 files changed, 60 insertions(+), 18 deletions(-)
> root@ubuntulocal:/var/www/html/ruTorrent# git apply --verbose --ignore-whitespace 97ae1e0dc579c3d0d5a185b4bc72666d4a8157c8.patch
> Checking patch plugins/rss/init.js...
> Applied patch plugins/rss/init.js cleanly.

Reply:
@Antorell Here's a correction, if you want to give it a try. I misread the documentation for the `Sanitizer` class. What this does is removes JavaScript injection from RSS feeds. It should allow us to display HTML text properly.
```
const sanitizer = new Sanitizer();
const doc = new DOMParser().parseFromString(String(data), 'text/html');
$("#rsslayout").setHTML(sanitizer.sanitize(doc.body)); 
```

Reply:
@stickz This will not work in most of cases.
https://caniuse.com/?search=Sanitizer

Reply:
Firefox 108 here, and it doesn't work still getting. I'm using Firefox 108. I tried Chromium edge, slightly different error but still an error. 

Firefox 108
> [28.12.2022 07:54:34] JS error: [http://192.168.1.226:643/ line 2 > injectedScript : 4066] ReferenceError: Sanitizer is not defined

Chromium Edge 108

> [28.12.2022 08:01:40] JS error: [http://192.168.1.226:643/ : 4066] Uncaught ReferenceError: Sanitizer is not defined
With 
```
rTorrentStub.prototype.getrssdetailsResponse = function(data)
{
    const sanitizer = new Sanitizer();
    const doc = new DOMParser().parseFromString(String(data), 'text/html');
    $("#rsslayout").setHTML(sanitizer.sanitize(doc.body)); 
    return(false);
}
```